source/main/webpack.config.js

```diff
@@ -22,6 +26,15 @@ module.exports = {
     __dirname: false,
     __filename: false
   },
+  plugins: [
```

Comment:
cc @nikolaglumac

Reply:
i believe this bug was introduced during the webpack3 changes, and it results in those variables not being embeded into the code on the main.js side, this fixes it